### DISCUSSION TITLE
Perhaps you could present at this event?!

### DISCUSSION THREAD

**Discussion by sugatoray** [clementchadebec/benchmark_VAE#50] [General]
Submission Deadline: 30-Sep-2022
How: In-Person

PyTorch Conference: 22-Dev-2022

Conference Details: https://pytorch.org/events?utm_source=twitter&utm_medium=organic_social&utm_campaign=conference_date

cc: @clementchadebec

![image](https://user-images.githubusercontent.com/10201242/187824048-d14e17ad-d265-46a9-a719-6c239f6a7626.gif)

![image](https://user-images.githubusercontent.com/10201242/187824212-87020de7-f5e8-479d-86af-57607da0e695.jpeg)

**Comment by clementchadebec** [comment]
Hi @sugatoray,
Thank you so much for sharing this! I will definitively have a look at this event :)
